$OpenBSD: patch-include_openh323buildopts_h_in,v 1.2 2013/03/02 14:06:24 ajacoutot Exp $

error: ptlib/../../revision.h: No such file or directory

XXX PTLIB_VER
http://lists.packetizer.com/pipermail/h323plus/2012-December/002543.html

--- include/openh323buildopts.h.in.orig	Fri Sep 21 05:53:21 2012
+++ include/openh323buildopts.h.in	Sun Feb 24 10:45:18 2013
@@ -33,9 +33,8 @@
 #define _OPENH323_BUILDOPTS_H
 
 #include <ptbuildopts.h>
-#include <ptlib/../../revision.h>
 
-#if PTLIB_MAJOR == 2 && PTLIB_MINOR < 10
+#if PTLIB_MAJOR == 2 && PTLIB_MINOR <= 10
    #define PTLIB_VER ( PTLIB_MAJOR*100 + PTLIB_MINOR*10 + PTLIB_BUILD )
 #else
    #define PTLIB_VER ( PTLIB_MAJOR*1000 + PTLIB_MINOR*10 + PTLIB_BUILD )
